lib/localStorage.ts

```diff
@@ -0,0 +1,87 @@
+// Collection of helper functions to interact with localStorage.
+// See original code in Frontend:
+// https://github.com/guardian/frontend/blob/master/static/src/javascripts/lib/storage.js
+const checkAvailability = (): boolean => {
+  const key = 'local-storage-module-test';
+  try {
+    localStorage.setItem(key, 'graun');
+    localStorage.removeItem(key);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+const storage = window.localStorage;
+const isAvailable = checkAvailability();
+
+export const get = (key: string) => {
```

Comment:
I'll try and lint this in a separate PR, but can we add return types to all functions (or at least all exported ones).

Reply:
I've added return types to all functions. This `get` function in particular is so generic that I ended up copying what Frontend does and use `any` instead of `string | object | array | null` which adds little no no value and creates more noise at the call site. Not sure there's anything else to lint but feel free to raise an issue you spot anything else.